Project/Source/Components/ComponentAudioListener.cpp

```diff
@@ -72,6 +79,26 @@ void ComponentAudioListener::UpdateAudioListener() {
 	};
 	alListenerfv(AL_POSITION, position.ptr());
 	alListenerfv(AL_ORIENTATION, orientation);
+	switch (distanceModel) {
+	case DistanceModel::EXPONENT:
+		if (clamped)
+			alDistanceModel(AL_EXPONENT_DISTANCE_CLAMPED);
+		else
+			alDistanceModel(AL_EXPONENT_DISTANCE);
+		break;
+	case DistanceModel::INVERSE:
+		if (clamped)
+			alDistanceModel(AL_INVERSE_DISTANCE_CLAMPED);
+		else
+			alDistanceModel(AL_INVERSE_DISTANCE);
+		break;
+	case DistanceModel::LINEAR:
+		if (clamped)
+			alDistanceModel(AL_LINEAR_DISTANCE_CLAMPED);
+		else
+			alDistanceModel(AL_EXPONENT_DISTANCE);

```

Comment:
If/else statements should always use braces

Reply:
braces included